test/actions/Test_Actions.php

```diff
@@ -1498,5 +1498,46 @@ public function cannot_set_empty_actionid()
         // An empty ActionID
         $action->setActionID('');
     }
+
+    /**
+     * @test
+     * @expectedException \PAMI\Exception\PAMIException
+     */
+    public function cannot_set_updateconfig()
```

Comment:
1. Logoff action call is a fuckup definetly. Fixed.
2. What about all "can" methods - in this particular action a number of a value in sequence is vital. UpdateConfig action demands a proper ordering. So I tryed to check this mechanics only, moreover it seems "set" family is common and tested and I have change nothing in setKey behavior. Am I wrong in this approach?